src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -209,7 +213,7 @@ ptree CARMAFreightPlugin::createMobilityOperationXml( ptree &json_payload ) {
 }
 
 
-CARMAFreightPlugin::PortDrayage_Object CARMAFreightPlugin::retrieveNextAction( std::string action_id ) {
+PortDrayagePlugin::PortDrayage_Object PortDrayagePlugin::retrieveNextAction( std::string action_id ) {
 	try{
 		// Set action_id
 		next_action_id->setString(1,action_id);
```

Comment:
Since the 'action_id' passed to this function is the previously completed action id by the CMV, maybe the name for this object could be previous_action_id or current_action_id rather than next_action_id?

Reply:
Additionally, if the 'next_action_id' object is renamed here, then the 'next_action' object on line 225 could be renamed to 'next_action_id' to maintain consistency.